data/json/items/comestibles/drink.json

```
@@ -1029,6 +1029,7 @@
     "name": { "str_sp": "clean water" },
     "description": "Fresh, clean water.  Truly the best thing to quench your thirst.",
     "color": "light_cyan",
+    "flags": [ "EATEN_COLD" ],
```

For `copy-from`ed items that need to delete one of the parent's flags, it's better to use `delete`.
```suggestion
    "delete": { "flags": [ "UNSAFE_CONSUME" ] },
```

---

will do, thx